<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Uwe Drechsel</author>
 <class>ExtraInfoDialog</class>
 <widget class="QDialog" name="ExtraInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VYM - Info</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item>
    <widget class="QLabel" name="textLabel1_2">
     <property name="text">
      <string>Map:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="mapNameLE">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 240, 241);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>File location:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="fileLocationLE">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 240, 241);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="accessibleDescription">
      <string extracomment="Label for map title"/>
     </property>
     <property name="text">
      <string>Title:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="mapTitleLE"/>
   </item>
   <item>
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>Author:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="authorLE"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="textLabel2">
     <property name="text">
      <string>Comment:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="commentTE"/>
   </item>
   <item>
    <widget class="QLabel" name="textLabel2_2">
     <property name="text">
      <string>Statistics:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="statsTE">
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 240, 241);</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>160</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>authorLE</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>mapNameLE</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>ExtraInfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ExtraInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
